README.md

```diff
@@ -11,6 +11,7 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5334/jors.309.svg)](https://doi.org/10.5334/jors.309)
 [![release](https://img.shields.io/github/v/release/pybamm-team/PyBaMM?color=yellow)](https://github.com/pybamm-team/PyBaMM/releases)
 [![black code style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+[![Powered by NumFOCUS](https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](http://numfocus.org)
```

Comment:
Should we move it to first position?

Reply:
Looks nice in the first position!

Reply:
Being superpicky on the badges, is there a way to have the contributors badge in the same row as the others without breaking the bot workflow? (I am worried that the bot will crash if we remove the empty lines)

Reply:
Yes, moving the contributors badge to the line with all the badges will make the bot ignore it. But, we can move the badge down to the [contributors](https://github.com/pybamm-team/PyBaMM#-contributors) section. We do have PyPI and conda related badges in the [installation](https://github.com/pybamm-team/PyBaMM#-installing-pybamm) section of the README.

Also, maybe we should have the "downloads" badge on the top?

Reply:
I quite like the contributors badge at top, but happy either way. It would be great to have some sort of "downloads metric" badge on top, I can't think of a good way around (two would be too much, and if it's just one it is arbitrary which one we choose).

Reply:
Yes, I guess the current configuration looks good for now. In the future, if the badge count increases, we can switch to a [table-like structure](https://github.com/sktime/sktime#welcome-to-sktime) - 

<img width="850" alt="image" src="https://user-images.githubusercontent.com/74055102/215074921-5224ea95-238c-4c57-afc4-c015c140e446.png">